### DISCUSSION TITLE
Error with the new version 2.0.0

### DISCUSSION THREAD

**Discussion by ghost** [nest-modules/mailer#1174] [General]
Hi everyone! before i used the version 1.11.2 of this package but today i update it to 2.0.0 and i have this error when i start the app `Error: Cannot find module 'preview-email'`

**Comment by KiwiKilian** [comment]
Please transform this to an issue. Seeing the same error

> Error: Cannot find module 'preview-email'

It originates from `/node_modules/@nestjs-modules/mailer/dist/mailer.service.js:8:22`, where it's a plain `require`:
```js
const previewEmail = require("preview-email");
```
Either the [new try/catch](https://github.com/nest-modules/mailer/commit/350688518e72d1fa347a11c2b1ed84988dbcc55a#diff-1cbc829f8e21eb159a441d9c2498ced6ff8461584a4e4d6a73dd837e202027e3R40-R44) block gets stripped by the build or an out-of-date `dist` build was published.
